controllers/replication-controller/persistentvolume_controller_test.go

```diff
@@ -473,6 +473,15 @@ func TestPVReconcileSuite(t *testing.T) {
 	suite.Run(t, testSuite)
 }
 
+func (suite *PVReconcileSuite) TestPVNotFound() {
+
+	req := suite.getTypicalRequest("no-pv")
+	_, err := suite.reconciler.Reconcile(context.Background(), req)
+	if err == nil {
+		suite.T().Log("No pv found")
```

Comment:
I believe we should be using assert.NoError(suite.T(), err, "No pv found"), otherwise I don't think the test will fail. It looks like other tests are using suite.NoError(...) to check it as well.

Reply:
sure I can update that and test it again